device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/DeviceCalendarPlugin.kt

```diff
@@ -96,6 +104,22 @@ class DeviceCalendarPlugin() : MethodCallHandler {
                 event.start = eventStart!!
                 event.end = eventEnd!!
 
+                if(call.hasArgument(RECURRENCE_RULE_ARGUMENT) && call.argument<Map<String, Any>>(RECURRENCE_RULE_ARGUMENT) != null) {
+                    val recurrenceRuleArgs = call.argument<Map<String, Any>>(RECURRENCE_RULE_ARGUMENT)!!
+                    val recurrenceFrequencyIndex = recurrenceRuleArgs[RECURRENCE_FREQUENCY_ARGUMENT] as Int
```

Comment:
What's going to happen if the `recurrenceRuleArgs[RECURRENCE_FREQUENCY_ARGUMENT]` is not an `Int`? Will this throw exceptions or will fallback to the default of `Int`?

Reply:
there'll be an exception

Reply:
Is this something we should watch out for?

Reply:
no the parameters come over via the platform channel like all the others so can expect the types to match what we've defined on the Dart side

Reply:
👍